src/builder/Actions.sol

```diff
@@ -122,6 +124,55 @@ library Actions {
         uint256 blockTimestamp;
     }
 
+    struct CometBorrowInput {
+        Accounts.ChainAccounts[] chainAccountsList;
+        uint256 amount;
+        string assetSymbol;
+        uint256 blockTimestamp;
+        address borrower;
+        uint256 chainId;
+        uint256[] collateralAmounts;
+        string[] collateralAssetSymbols;
+        address comet;
+    }
+
+    struct CometRepayInput {
+        Accounts.ChainAccounts[] chainAccountsList;
+        uint256 amount;
+        string assetSymbol;
+        uint256 blockTimestamp;
+        uint256 chainId;
+        uint256[] collateralAmounts;
+        string[] collateralAssetSymbols;
+        address comet;
+        address repayer;
+    }
+
+    struct MorphoBorrow {
+        Accounts.ChainAccounts[] chainAccountsList;
+        string assetSymbol;
+        uint256 amount;
+        uint256 chainId;
+        address borrower;
+        uint256 blockTimestamp;
+        uint256 collateralAmount;
+        string collateralAssetSymbol;
+    }
+
+    struct MorphoRepay {
+        Accounts.ChainAccounts[] chainAccountsList;
+        string assetSymbol;
+        uint256 amount;
+        uint256 chainId;
+        address repayer;
+        uint256 blockTimestamp;
+        uint256 collateralAmount;
+        string collateralAssetSymbol;
+        // Needed for handling max repay
+        bool payInShares;
+        uint256 borrowedShares;
+    }
```

Comment:
nit: These fields should be alphabetically ordered

Reply:
Sure I will reordered those fields. 
iirc I copied the layout from one of the prior structs... (likely those were not ordered either 😄 )